HousingRepairsOnlineApi.Tests/HelpersTests/RepairBookingResponseHelperTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.Helpers;
+using Moq;
+using Xunit;
+
+namespace HousingRepairsOnlineApi.Tests.HelpersTests
+{
+    public class RepairBookingResponseHelperTests
+    {
+        private readonly RepairBookingResponseHelper _systemUnderTest;
```

Comment:
Remove leading underscores '_' (from this and others).

Is this repo configured to have/suggest leading underscores? If so, that should be changed (doesn't need to be completed in this PR though).

Reply:
I get Rider complaining but I'll change them back to be consistent. Not sure if it's an IDE setting that's causing it to complain. I'll have a look.